src/toil/jobStores/azureJobStore.py

```diff
@@ -120,8 +120,45 @@ def qualify(self, name):
         return self.namePrefix + self.nameSeparator + name
 
     def jobs(self):
-        for jobEntity in self.jobItems.query_entities():
-            yield AzureJob.fromEntity(jobEntity)
+        # We need to page through the results, since we only get some of them at
+        # a time. Just like in the BlobService. See the only documentation
+        # available: the API bindings source code, at:
+        # https://github.com/Azure/azure-storage-python/blob/09e9f186740407672777d6cb6646c33a2273e1a8/azure/storage/table/tableservice.py#L385
+        
+        # These two together constitute the primary key for an item.
+        next_partition_key = None
+        next_row_key = None
+    
+        # How many jobs have we done?
+        total_processed = 0
+    
+        while True:
+            # Get a page (up to 1000 items)
+            page = self.jobItems.query_entities(
```

Comment:
Can the paging functionality be placed in a reusable method of the AzureTable class?